.evergreen/config.yml

```diff
@@ -1436,11 +1436,29 @@ tasks:
       commands:
         - func: "bootstrap mongo-orchestration"
           vars:
-            VERSION: "latest"
+            VERSION: "5.0"
             TOPOLOGY: "server"
         - func: "run mod_wsgi tests"
 
     - name: "mod-wsgi-replica-set"
+      tags: ["mod_wsgi"]
+      commands:
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            VERSION: "5.0"
+            TOPOLOGY: "replica_set"
+        - func: "run mod_wsgi tests"
+
+    - name: "mod-wsgi-standalone-latest"
```

Comment:
I think we can stop test latest altogether on the failing platforms in this PR. It's not worth trying to migrate.

Reply:
Done.  Load balancer tests are now [passing](https://spruce.mongodb.com/version/63517dcd3627e05bababdce9/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

Reply:
Also mod_wsgi with 5.0